src/main/java/jfdi/parser/Constants.java

```diff
@@ -0,0 +1,97 @@
+package jfdi.parser;
+
+import java.time.ZoneId;
+
+public class Constants {
+
+    // ===============================
+    // Regex Constants
+    // ===============================
+
+    // General Regex Strings
+    public static final String REGEX_WHITESPACE = "\\s";
+
+    // Task type specific Regex Strings
+    public static final String REGEX_ADD = "^(add)";
+    public static final String REGEX_LIST = "^(list)";
+    public static final String REGEX_EDIT = "^(edit)";
+    public static final String REGEX_DELETE = "^(delete)";
+
+    // Task related Regex Strings
+    // Tags are always appended to the end of any input, hence the boundary
+    // condition $
+    public static final String REGEX_TAGS = "\\+\\w+$";
```

Comment:
Yes it will. In AddCommandParser, I strip the input starting from the rear, tag by tag, so every tag is taken as the end of the continually stripped input.